build/deps/build_deps.jsonc

```diff
@@ -167,6 +167,12 @@
       "name": "com_google_benchmark",
       "owner": "google",
       "repo": "benchmark"
+    },
```

Comment:
Would you mind removing the previous "com_google_benchmark" named dependency?

Reply:
I removed it